templates/cluster-template-cilium.yaml

```diff
@@ -77,7 +77,6 @@ spec:
       rootVolume:
         id: root
         image: "${MVM_ROOT_IMAGE:=ghcr.io/weaveworks-liquidmetal/capmvm-kubernetes:1.21.8}"
-        mountPoint: "/"
```

Comment:
this is just embarrassing, i thought i had removed all these before the last release. i guess not! i have updated the templates on the release 🤦‍♀️